adopt-a-dog/dogs.html

```diff
@@ -0,0 +1,233 @@
+<!DOCTYPE html>
+<html lang="zxx">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="stylesheet.css">
+    <script src="javascript.js"></script>
+    <title>ADOPTADOG</title>
+</head>
+<body>
+    <header>
+      <div class="banner">
+        <div class="headline">
+          <img src="adoptadog-logo.svg" alt="site-logo" width=300>
+          <h1>Seattle's premiere dog adoption agency</h1>
+        </div>
+        <div class="check-out">
+          <img src="cart-icon.svg" alt="shopping cart icon" width=30>
+          <form action="#" method="post">
+            <input type="button" name="checkout" value="Checkout">
+          </form>
+        </div>
+      </div>
+      <nav>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="dogs.html">Dogs</a></li>
+          <li><a href="blog.html">Blog</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <h2>Here are all of our Dogs!</h2>
+      <div class="dog-container">
+        <article class="dog-profile">
+          <div>
+            <img src="images/alex-lacayo-756508-unsplash.jpg" alt="photo of Murphy" width=200 onclick="dogDetail('Murphy','Retriever mix', '$123.45');">
```

Comment:
Please move the width to your css file, I see in a couple of places. ALL css should be in the file, not in the HTML